client/src/components/main_section/main_section.coffee

```diff
@@ -12,7 +13,7 @@ SingleRepo = React.createFactory require '../repo_list/single_repo.coffee'
 
 MainSection = React.createClass
   render: ->
-    {repo, repoActions, login, pageLocation, pageLocationActions,loginActions} = @props
+    {repo, repoActions, login, pageLocation, pageLocationActions,loginActions, registerActions} = @props
```

Comment:
I just said this below aha. Yeah makes sense. Though we can clean it up by merging a props object with a object of bound ActionCreators. http://rackt.github.io/redux/docs/api/bindActionCreators.html